### PR TITLE
Fix ops flag mask master

### PR DESCRIPTION
**Pull request by fmarco76**
@edewata  this PR add two flags `pki_*_opsFlag` and `pki_*_opsFlagMask`. These have been introduced in branch v10.13 with PRs #4643, #4645 and #4647 but the code in master is too different for cherry-picking so I have written from scratch and added test and doc.

### CODE REVIEW

**Review by edewata — APPROVED** [review]
Feel free to merge since the functionality has been added into other branches, but please see my comments & questions below. We can address them in separate PRs if needed.

File: base/tools/src/main/java/com/netscape/cmstools/nss/NSSKeyCreateCLI.java [hunk 1]
```diff
@@ -86,6 +86,14 @@ public void createOptions() {
         option.setArgName("boolean");
         options.addOption(option);
 
+        option = new Option(null, "ops-flag", true, "Custom flags for key usage (empty for HSM default)");
```
[the remark this answers is not in the document]
Comment (fmarco76) [reply base/tools/src/main/java/com/netscape/cmstools/nss/NSSKeyCreateCLI.java:89]: Not sure about the wording but your clarification seems reasonable so I'll update.

File: .github/workflows/ca-hsm-operation-test.yml [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+              -D pki_token_name=HSM \
+              -D pki_token_password=Secret.HSM \
+              -D pki_server_database_password=Secret.123 \
+              -D pki_ca_signing_token=HSM \
+              -D pki_ocsp_signing_token=HSM \
+              -D pki_audit_signing_token=HSM \
+              -D pki_subsystem_token=HSM \
+              -D pki_sslserver_token=internal \
+              -D pki_ca_signing_opsFlag=sign \
+              -D pki_ca_signing_opsFlagMask=sign \
```
[the remark this answers is not in the document]
Comment (fmarco76) [reply .github/workflows/ca-hsm-operation-test.yml:112]: opsFlagMask is applied to the flags identified by NSS, then the opsFlag are added to the remaining operation. This test show that the opsFlag are added to the resulting list. (I will change all opsFlag* is opFlags*)

[the remark this answers is not in the document]
Comment (edewata) [reply .github/workflows/ca-hsm-operation-test.yml]: OK, does it mean the `opsFlag` param overrides `opsFlagMask`? Does it work that way too with `--keyOpFlagsOn` and `--keyOpFlagsOff` in `certutil`? We probably should document this behavior.

**Comment by fmarco76** [comment]
@edewata Thanks! I have updated the code following your comments.

**Pull request merged**